docs/topics/azure-stack.md

```diff
@@ -149,6 +149,14 @@ If you need to expose more than 5 services, then the recommendation is to route
 
 ## Frequently Asked Questions
 
-### Network Policies
+### Network policies enforcement
 
-To enforce network policies, you are required to manually deploy the [Canal](https://docs.projectcalico.org/v3.7/getting-started/kubernetes/installation/flannel) daemonset.
\ No newline at end of file
+To enforce network policies, you are required to manually deploy the [Canal](https://docs.projectcalico.org/v3.7/getting-started/kubernetes/installation/flannel) daemonset.
+
+### Sample extensions are not working
+
+Extensions in AKS Engine provide an easy way to include your own customization at provisioning time.
+
+Because Azure and Azure Stack currently rely on a different version of the Compute Resource Provider API, you may find that some of sample [extensions](https://github.com/Azure/aks-engine/tree/master/extensions) fail to deploy correctly.
+
+This can be resolved by making a small modification to the extension `template.json` file. Replacing all usages of template parameter `apiVersionDeployments` by the hard-code value `2017-12-01` (or whatever API version Azure Stack runs at the time you try to deploy) should be all you need.
```

Comment:
the extensions get fetched from https://raw.githubusercontent.com/Azure/aks-engine/master/ by default, do you suggest the user modifies the template on their own branch and set the `rootURL` to point to their branch?

Reply:
pushed clarification